packages/@azure/eventhubs/processor/tests/eph.spec.ts

```diff
@@ -477,6 +477,8 @@ describe("EPH", function (): void {
           await host.getPartitionInformation("");
         } catch (err) {
           err.message.should.match(/.*The specified partition is invalid for an EventHub partition sender or receiver.*/ig);
+        } finally {
+          await host.stop();
```

Comment:
Missing `host.stop()` didn't cause problem but I think it doesn't hurt to add it.